packages/fixie/tests/fixtures/test-fixie-config.yaml

```diff
@@ -0,0 +1,2 @@
+apiUrl: https://fake.api.domain
```

Comment:
curious why we need to have apiUrl duplicated across this file and in `test-fixie-config-ignore-fields.yaml`

Reply:
They are two different files used in two different tests. One of them tests what happens when the file is nominal, i.e., no bad or unknown fields, the other has the additional unknown field in it.